src/Components/Table/Rows.js

```diff
@@ -2,26 +2,26 @@ import React from "react";
 import moment from "moment";
 
 const Rows = props => {
+  const checkIn = moment(props.data.checkInDate);
+  const checkOut = moment(props.data.checkOutDate);
   return (
-    <tbody>
-      {props.fakeData.map((booking, index) => {
-        const checkIn = moment(booking.checkInDate);
-        const checkOut = moment(booking.checkOutDate);
-        return (
-          <tr key={index}>
-            <th scope="row">{booking.id}</th>
-            <td>{booking.title}</td>
-            <td>{booking.firstName}</td>
-            <td>{booking.surname}</td>
-            <td>{booking.email}</td>
-            <td>{booking.roomId}</td>
-            <td>{booking.checkInDate}</td>
-            <td>{booking.checkOutDate}</td>
-            <td className="booking-days">{checkOut.diff(checkIn, "days")}</td>
-          </tr>
-        );
-      })}
-    </tbody>
+    <tr
+      key={props.data.id}
+      className={props.isSelected ? "select" : "notSelect"}
+      onClick={() => {
+        props.setSelectedRow(props.data.id);
```

Comment:
See my comment above about how we could make selecting rows independent from each other. You're really close! 👍

Reply:
I change it! Thank you !